README.md

```diff
@@ -1,8 +1,34 @@
 # phoenix
 
-On-chain orderbook that operates without a crank
+Phoenix is an on-chain orderbook that operates without a crank.
 
-### Build
+### Documentation
+
+Documentation and instructions on how to interact with the program are available on [GitBook](https://ellipsis-labs.gitbook.io/phoenix-dex/tRIkEFlLUzWK9uKO3W2V/getting-started/phoenix-overview).
+
+### Licensing
+
+The primary license for Phoenix is the Business Source License 1.1 (`BUSL-1.1`), which can be found at [`LICENSE`](https://github.com/Ellipsis-Labs/phoenix-v1/blob/master/LICENSE).
+
+### Audits
+
+Phoenix has been audited by OtterSec. The audit report can be found at [audits/OtterSec.pdf](https://github.com/Ellipsis-Labs/phoenix-v1/blob/master/audits/OtterSec.pdf).
+
+### Bug Bounty
+
+Information on the bug bounty program for Phoenix can be found at [SECURITY.md](https://github.com/Ellipsis-Labs/phoenix-v1/blob/master/SECURITY.md).
+
+### Build Verification
+
+You can use [Solana Verify CLI](https://github.com/Ellipsis-Labs/solana-verifiable-build) to verify that the program deployed at `PhoeNiXZ8ByJGLkxNfZRnkUfjvmuYqLR89jjFHGqdXY` matches the code in this repository. After installing the CLI, run:
+
+```
+solana-verify verify-from-repo -um --program-id PhoeNiXZ8ByJGLkxNfZRnkUfjvmuYqLR89jjFHGqdXY https://github.com/Ellipsis-Labs/phoenix-v1
```

Comment:
I think we want to specify a commit hash here. @throwbackjams can help

Reply:
shouldn't we just add that only if we change the build?

the README shouldn't be the place we publish the hash